### ISSUE TITLE
join HydroShare link took me back to index with url http://beta.hydroshare.org/?next=

### ISSUE THREAD

**Issue by mtpain** [hydroshare/hydroshare#184]
After I clicked on the "SIGN IN" tab I tried to click 'join HydroShare', but was just redirected to the main splash page.

**Comment by mjstealey** [comment]
@mtpain - The main splash page contains the minimal required elements to join HydroShare {First Name, Last Name, Email, Username, Password}. Once the required fields are filled in and the "Join HydroShare & start collaborating" button is pressed a verification email will be sent to the email address provided in the form.

![joinhs](https://cloud.githubusercontent.com/assets/5332509/6430834/d07d557c-bfe9-11e4-9c06-95690c54b086.png)

The behavior you've denoted is the expected behavior of the site. Hopefully this makes sense.

**Comment by mjstealey** [comment]
Is presently the expected behavior of HydroShare

**Comment by mtpain** [comment]
This does make sense.

I should have posted a screenshot as well. I wasn't seeing the registration form before. Thanks!

**Comment by mohamedmorsy** [comment]
Hi Michael,

Now, I am working on my modifications for Model Instance resource on the
new branch model-instance-update. During test the resource parsed metadata
included in the downloaded parcel (resourcemetdata.xml), I found that the
temporal coverage metadata gives the wrong date which is constant for any
created resource even if it is generic resource or model instance resource.
The value of the temporal coverage
is <rdf:value>start=2015-01-01T00:00:00+00:00;
end=2015-01-01T00:00:00+00:00; scheme=W3C-DTF</rdf:value> if the input
dates in 2015 while it is <rdf:value>start=2014-01-01T00:00:00+00:00;
end=2015-01-01T00:00:00+00:00; scheme=W3C-DTF</rdf:value> if the input
dates in 2014 and 2015 while it appears correct on the website. I tested
that on my local machine and also on the dev.hydroshare.org website. Here
are two screen shots for more illustration:

[image: Inline image 2]

[image: Inline image 1]

Thanks,

Mohamed

On Mon, Mar 2, 2015 at 5:38 PM, Matthew <notifications@github.com> wrote:

> This does make sense.
>
> I should have posted a screenshot as well. I wasn't seeing the
> registration form before. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hydroshare/hydroshare/issues/184#issuecomment-76842446>
> .
>



-- 
Mohamed M. Morsy
Graduate Research Assistant
Department of Civil & Environmental Engineering
Thornton Hall D107, University of Virginia
351 McCormick Road, P.O. Box 400742
Charlottesville, VA 22904-4742
Phone: (434) 924-7630
E-mail: mmm4dh@virginia.edu <alamm@email.sc.edu>
